backend/src/socket/trace/post.trace.ts

```diff
@@ -138,6 +138,21 @@ const commentAdd = async (
   return dalTrace.create(trace);
 };
 
+const commentRemove = async (
+  input: SocketPayload<CommentModel>,
+  eventType: string
+) => {
+  const trace = await createTrace(input.trace);
+  const comment = input.eventData;
+  trace.event = {
+    postID: comment.postID,
+    commentID: comment.commentID,
+    commentModifiedText: comment.comment,
```

Comment:
This shouldn't be a field for the comment remove event. It should be something similar to the event for removing a post, so just one field like "commentDeleted".

Reply:
I kept the commentID field and added a commentDeleted to make it like the post remove event